index.html

```diff
@@ -2579,9 +2588,16 @@
             <code>String</code>, return <code>false</code> and abort these steps.
           </li>
           <li>
+            Let <code>subdomain_pattern</code> be the result of prepending <code>"."</code>
```

Comment:
Would it compile :smile:? Do you want me to remove '_' separators from the algorithm?